src/components/change-requests/change-request-details/change-request-details/review-notes/review-notes.test.tsx

```diff
@@ -67,19 +68,19 @@ describe('Change request review notes test', () => {
 
   it('renders no date tooltip on hover', async () => {
     renderComponent(cr[2]);
-    fireEvent.mouseOver(screen.getByText(fullNamePipe(exampleAppAdminUser)));
+    fireEvent.mouseOver(screen.getByText(fullNamePipe(exampleStageGateChangeRequest.reviewer)));
 
-    expect(await screen.findByText('Reviewed On: —')).toBeInTheDocument();
+    expect(await screen.findByText('Reviewed on: —')).toBeInTheDocument();
```

Comment:
Is this test is a necessary **test** to pass? For some reason, only this test is not passing but the application, itself works fine. 

Application Screenshot:
![NER_#480_Tool_Tip_Test_ScreenShot](https://user-images.githubusercontent.com/91857946/156907513-9b3d0d72-8b81-4502-8cf6-9de64e86fc26.png)

Codebase (Test) Screenshot: (cr[2] is exampleStageGateChangeRequest)
![NER_#480_Tool_Tip_CodeBaseTest_screenShot](https://user-images.githubusercontent.com/91857946/156907569-7a323033-89ce-4305-82ba-cde3e7b854a7.png)

Reply:
Is the test not working?

Reply:
Yes, but the application, itself is working fine. Like in the change request web page, it shows 'Reviewed on: -' as in the first screenshot.

Reply:
I walked through this test issue with James today during office hours, and we were able to resolve it! We just had to modify the test a little bit and all the tests are passing now.